### PR TITLE
Adding issue callout to Readme

### PR DESCRIPTION
**Pull request by BobaFetters**
Adding a callout for reporting issues in the README for `apollo-ios-codegen` and `apollo-ios-dev` as suggested in feedback in apollographql/apollo-ios#3240

### CODE REVIEW

File: apollo-ios-codegen/README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ This repo provides the code necessary to do GraphQL code generation for the [Apo
 
 However, if you plan to handle your code generation through Swift scripting, you will now need to include the `apollo-ios-codegen` package as a dependency as it is no longer packaged as part of `apollo-ios`. In order to get start with scripting your code generation check out our guide [here](https://www.apollographql.com/docs/ios/code-generation/run-codegen-in-swift-code).
 
+## Issues
+
+To report an issue, bug, or feature request ou can do so in the [apollo-ios](https://github.com/apollographql/apollo-ios) repo.
```
Comment (calvincestari) [inline apollo-ios-codegen/README.md:34]: Same comment as above.

File: README.md [hunk 2]
```diff
@@ -53,13 +53,17 @@ After you have run the `tuist generate` command you should see both an `ApolloDe
 
 After working and making changes in the `ApolloDev.xcworkspace` you can commit your changes as normal and submit a PR to the `main` branch of the `apollo-ios-dev` repo for review.
 
+## Issues
+
+To report an issue, bug, or feature request ou can do so in the [apollo-ios](https://github.com/apollographql/apollo-ios) repo.
```
Comment (calvincestari) [inline README.md:58]: Should this link instead go directly to the issues list?

File: README.md [hunk 3]
```diff
@@ -83,4 +87,4 @@ For further information on contributing, reporting issues, suggesting features,
 
 ## Learn how to build with Apollo
 
-Check out the [Odyssey](https://odyssey.apollographql.com/) learning platform, the perfect place to start your GraphQL journey with videos and interactive code challenges. Join the [Apollo Community](https://community.apollographql.com/) to interact with and get technical help from the GraphQL community.
\ No newline at end of file
+Check out the [Odyssey](https://odyssey.apollographql.com/) learning platform, the perfect place to start your GraphQL journey with videos and interactive code challenges. Join the [Apollo Community](https://community.apollographql.com/) to interact with and get technical help from the GraphQL community.
```
Comment (calvincestari) [inline README.md:90]: I'm not sure what the change is here?

**Review by calvincestari — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (BobaFetters) [reply README.md:90]: Yea I'm not sure why this is marked as a change because nothing is changed except line number.

**Pull request merged**